release/rancher/rancher.go

```diff
@@ -27,50 +27,69 @@ const (
 	rancherImagesFileName    = "/rancher-images.txt"
 	rancherHelmRepositoryURL = "https://releases.rancher.com/server-charts/latest/index.yaml"
 
-	setKDMBranchReferencesScriptFile = "set_kdm_branch_references.sh"
-	setKDMBranchReferencesScript     = `#!/bin/bash
+	setKDMBranchReferencesScriptFileName = "set_kdm_branch_references.sh"
+	setChartReferencesScriptFileName     = `set_chart_references_script.sh`
+	cloneCheckoutRancherScript           = `#!/bin/bash
```

Comment:
Can you run an example of this script through shellcheck and remediate anything it finds?